pkg/ebpf/utils_linux.go

```diff
@@ -105,22 +100,20 @@ func verifyOSVersion(kernelCode uint32, platform string, exclusionList []string)
 	}
 
 	// using eBPF causes kernel panic for linux kernel version 4.4.114 ~ 4.4.127
-	if isLinuxAWSUbuntu(platform) || isUbuntu(platform) {
-		if kernelCode >= linuxKernelVersionCode(4, 4, 114) && kernelCode <= linuxKernelVersionCode(4, 4, 127) {
-			return false, fmt.Sprintf("Known bug for kernel %s on platform %s, see: \n- https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1763454", kernelCodeToString(kernelCode), platform)
-		}
+	if platform == "ubuntu" && kernelCode >= kernel.VersionCode(4, 4, 114) && kernelCode <= kernel.VersionCode(4, 4, 127) {
+		return false, fmt.Sprintf("Known bug for kernel %s on platform %s, see: \n- https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1763454", kernelCode, platform)
 	}
```

Comment:
Its not clear to be why we don't need strings.Contains(`ubuntu`) anymore - could you clarify?

Reply:
The `platform` value that comes from the metadata package is more tightly controlled, so we can rely on the exact value `ubuntu`.

Reply:
Thanks! 🙇